CHANGELOG.next.asciidoc

```diff
@@ -9,6 +9,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 ==== Breaking changes
 
 *Affecting all Beats*
+-  Introduce log message for not supported annotations for Hints based autodiscover {pull}38213[38213]
```

Comment:
Is this a breaking change?

Reply:
Thanks nice catch. Updated in https://github.com/elastic/beats/pull/38213/commits/dbd20e5c67e180ac0bf48a48424e524e376506c1